src/sentry/runner/commands/createuser.py

```diff
@@ -37,6 +37,14 @@ def _get_superuser():
     return click.confirm("Should this user be a superuser?", default=False)
 
 
+def _set_user_permissions(user):
+    from sentry.models import UserRole, UserRoleUser
+
+    if click.confirm("Should this user have Super Admin permissions?", default=False):
```

Comment:
probably worth adding a quick description of Super Admin permissions:

> Should this superuser have Super Admin permissions? This allows the superuser to add / remove / edit users

Reply:
iu would probably just say "This grants them all permissions available". at least more explicit than just user management